packages/contracts-bedrock/deploy-config/sepolia.json

```diff
@@ -0,0 +1,41 @@
+{
+  "finalSystemOwner": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "portalGuardian": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "l1StartingBlockTag": "0x7c83e929deffe1b6a97cc9f9ef99845e6970a512b9ce18432d95d6f7f851ea53",
+  "l1ChainID": 11155111,
+  "l2ChainID": 11155420,
+  "l2BlockTime": 2,
+  "maxSequencerDrift": 600,
+  "sequencerWindowSize": 3600,
+  "channelTimeout": 300,
+  "p2pSequencerAddress": "0x715b7219D986641DF9eFd9C7Ef01218D528e19ec",
+  "batchInboxAddress": "0xff00000000000000000000000000000011155420",
+  "batchSenderAddress": "0x7431310e026B69BFC676C0013E12A1A11411EEc9",
+  "l2OutputOracleSubmissionInterval": 120,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleProposer": "0x02b1786A85Ec3f71fBbBa46507780dB7cF9014f6",
+  "l2OutputOracleChallenger": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "finalizationPeriodSeconds": 12,
+  "proxyAdminOwner": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "baseFeeVaultRecipient": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "l1FeeVaultRecipient": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "sequencerFeeVaultRecipient": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "l1FeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "sequencerFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "baseFeeVaultWithdrawalNetwork": 0,
+  "l1FeeVaultWithdrawalNetwork": 0,
+  "sequencerFeeVaultWithdrawalNetwork": 0,
+  "gasPriceOracleOverhead": 188,
+  "gasPriceOracleScalar": 684000,
+  "enableGovernance": true,
+  "governanceTokenSymbol": "OP",
+  "governanceTokenName": "Optimism",
+  "governanceTokenOwner": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+  "l2GenesisBlockGasLimit": "0x17D7840",
```

Comment:
Should be `0x1C9C380` - 30m max gas limit, elasticity of 6, which gives us a target of 5m gas.